AppProject/Client/Pages/Homepage.razor.cs

```diff
@@ -0,0 +1,27 @@
+﻿using AppProject.Client.Components;
+using AppProject.Shared;
+
+namespace AppProject.Client.Pages
+{
+    public partial class Homepage : BasePage
+    {
+        private IEnumerable<Item> items = Enumerable.Empty<Item>();
+
+        protected override async Task OnInitializedAsync()
+        {
+            await LoadItemsAsync().ConfigureAwait(false);
+        }
+        private async Task LoadItemsAsync()
+            => items = await Service.GetItemsAsync(CancellationToken).ConfigureAwait(false) ?? Enumerable.Empty<Item>();
```

Comment:
Wir sollten rein wegen der Logik wegen aus solchen Methoden GetItemsAsync machen und die Items zurückgeben. In der OnInitializedAsync dann das Zuweisen. Das macht zwar keinen großen Unterschied aber ist von dem Code meiner Meinung nach besser.